olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -426,6 +426,8 @@ spec:
                         value: "deploy/resources/dashboards/grafana-dash-knative-eventing-source.yaml"
                       - name: EVENTING_BROKER_DASHBOARD_MANIFEST_PATH
                         value: "deploy/resources/dashboards/grafana-dash-knative-eventing-broker.yaml"
+                      - name: EVENTING_CHANNEL_DASHBOARD_MANIFEST_PATH
```

Comment:
driveby comment: Is there a reason we don't just read and apply the entire `dashboards` folder? If we need scoping between serving and eventing etc, we could add subfolders and thelike. Maybe food for a followup.

Reply:
Yes no reason to install an eventin dashboard if eventing is never installed. The lifecycle of a dashboard depends on the instance of the corresponding CR (serving, eventing etc).